contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from future import standard_library
+standard_library.install_aliases()
+import json, urllib.request, urllib.parse, urllib.error
+class SpotifyHandler(object):
+    '''
+    This plugin will allow users to directly search
+    for song, artists, and albums using the Spotify API.
+    It can be called using `@spotify song/artist/album`
```

Comment:
This description wasn't very clear at the beginning - I had a few tries before I got that I'm supposed to put `song`, `artist` or `album` before the keywords. 

Can we have a general search query that would return objects of different types?

If not, why did you choose those three endpoints - are those the only ones available through the API?

I'd like this description to be more clear - I want the user to know that `artist` would give them 10 songs for the given artist.

Also, what should I provide if I already know the artist and the song title? This is not clear from the description.